gomock/controller.go

```diff
@@ -80,7 +83,7 @@ func NewController(t TestReporter) *Controller {
 	}
 }
 
-func (ctrl *Controller) RecordCall(receiver interface{}, method string, args ...interface{}) *Call {
+func (ctrl *Controller) RecordCall(receiver interface{}, method string, methodType reflect.Type, args ...interface{}) *Call {
```

Comment:
That's a good suggestion. I'm still leaning towards breaking backwards compatibility here because I think that only the mock and recorder should be calling these functions and the mock won't be using the older version of RecordCall.